### ISSUE TITLE
✨ Add a card advertising the IAP on talent search page when "IT-01" is selected

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#12085]
## ✨ Feature

Add a card to the "find talent" page that shows up at the top of the list of pools if "IT-01" is selected.

![image](https://github.com/user-attachments/assets/068bf02e-4d6c-47ff-9322-4e68ffbfd9ca)

### English copy
**Have you considered hiring an Indigenous IT Apprentice?**
The IT Apprenticeship Program for Indigenous Peoples aims to help address and remove barriers Indigenous peoples face when it comes to finding employment in the Government of Canada's digital workforce. Become a hiring partner today and discover how you can strengthen your team with Indigenous IT talent and help build a more inclusive public service. Your participation contributes to the broader goal of inclusion, equity, and reconciliation in Canada.

**Contact the team** 
This link should be the same as the one on https://talent.canada.ca/en/indigenous-it-apprentice/hire ... but my browser isn't letting me copy/paste the subject and body included for that email link.

**[Visit the IT Apprenticeship Program for Indigenous Peoples manager page](https://talent.canada.ca/en/indigenous-it-apprentice/hire)**

### French copy
**Pensez-vous embaucher une apprentie ou un apprenti en TI autochtone?**
Le Programme d'apprentissage en TI pour les personnes autochtones a pour but d'aider à éliminer les obstacles en matière d'emploi pour les personnes autochtones dans le domaine numérique au sein du gouvernement du Canada. Devenez un partenaire de recrutement dès aujourd'hui et découvrez comment vous pouvez renforcer votre équipe en embauchant des talents autochtones en informatique et aider à bâtir à une fonction publique plus inclusive. Votre participation contribue à l'objectif plus large de promouvoir l'inclusion, l'équité et la réconciliation au Canada.

**Communiquez avec l’équipe** 
This link should be the same as the one on https://talent.canada.ca/en/indigenous-it-apprentice/hire ... but my browser isn't letting me copy/paste the subject and body included for that email link.

**[Visitez la page du gestionnaire du Programme d'apprentissage en TI pour les personnes autochtones](https://talent.canada.ca/fr/indigenous-it-apprentice/hire)**

## ✅ Acceptance Criteria

- [ ] new card appears if IT-01 is selected in talent search page
- [ ] fine if it only appears when other search results appear
- [ ] appears before other search results

## 🛑 Blockers

```[tasklist]
### Blocked By
```

**Comment by mnigh** [comment]
> **Contact the team** This link should be the same as the one on https://talent.canada.ca/en/indigenous-it-apprentice/hire ... but my browser isn't letting me copy/paste the subject and body included for that email link.

@gobyrne there are two **Contact the team** links on that page that use the same email address but have different subjects. Is this the one you are referring to?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/d318276eef98becb52c2275f880292b58543012d/apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx#L47-L85

**Comment by tristan-orourke** [comment]
Yes Matt, that's the Contact link we want to use.
